packages/components/modules/profiles/web/ProfileSettingsComponent/index.tsx

```
@@ -0,0 +1,278 @@
+  const hasUploadedImage = imageUrl.includes(process.env.NEXT_PUBLIC_REMOTE_PATTERNS_HOSTNAME ?? '')
+  const hasUploadedBannerImage = bannerImageUrl.includes(
+    process.env.NEXT_PUBLIC_REMOTE_PATTERNS_HOSTNAME ?? '',
+  )
+
+  const onSubmit = async (data: ProfileUpdateForm) => {
+    const dirtyValues = filterDirtyValues({ values: data, dirtyFields })
+    const { id, image, bannerImage } = data
+    const uploadables: UploadablesObj = {}
+    if ('image' in dirtyValues && image && typeof image !== 'string') {
+      uploadables.image = image
+      delete dirtyValues.image
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)